### DISCUSSION TITLE
Ivalid Version while install hanldebars

### DISCUSSION THREAD

**Discussion by strelov1** [yarnpkg/berry#3875] [Q&A] [answered]
How can I ignore this error while yarn install ?
![image](https://user-images.githubusercontent.com/10761735/145945742-d92cd789-3b0d-4056-af0b-b4e665382e57.png)

yarn -v // 2.4.3

**Comment by arcanis** [comment, marked as answer]
You can't. Packages must be properly defined, and it seems one of yours is broken (invalid version number format, doesn't respect the semver rules).

**Reply by strelov1** [reply]
Thanks, I understood. Original packages have already fixed this problem, but broken packages still have in my mirror cache registry.
